test/SlowTests/Issues/RavenDB_21263.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
```

Comment:
lets assert the database record version as well in all those tests

Reply:
done